hitch.man.rst

```diff
@@ -50,6 +50,7 @@ Command line arguments
   -n  --workers=NUM          Number of worker processes (Default: 1)
   -B  --backlog=NUM          Set listen backlog size (Default: 100)
   -k  --keepalive=SECS       TCP keepalive on client socket (Default: 3600)
+  -R  --backendrefresh=SECS  Periodic backend IP lookup, 0 to disable (Default: 0)
```

Comment:
I suggest `--backend-refresh` to be consistent with the `backend-refresh` configuration parameter just like `prefer-server-ciphers`, `write-proxy*` et al. do.